google/cloud/paymentgateway/acquirerswitch/README.md

```diff
@@ -0,0 +1,8 @@
+# Google Cloud Acquirer Switch APIs
+
+
+UPI App developers can use Acquirer Switch proto files to generate the clients for invoking Acquirer switch APIs over gRPC/REST.
+
+## License
+
+Apache 2.0; see [`LICENSE`](LICENSE) for details.
```

Comment:
We will need to update this link to refer to root directory LICENSE file. I think this should work /LICENSE
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files